docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,151 @@
+
+Open the **Failover Cluster Manager** console on one of the nodes. If there is no cluster configured yet, you can right click **Failover Cluster Manager** amd select **New Cluster**
+
+On the **Select Servers** page, enter the Fully Qualified Domain Name of each node that will be in this cluster.
+
+On the **Validation Warning** page, select "Yes, When I click Next, run configuration validation tests, and then return to the process of creating the cluster"
+
+The **Validate a Configuration Wizard** appears, select **Run all Tests** and select **Next**
+
+When all validation appears OK, you will be returned to the **Create Cluster Wizard** where the **Access Point for Administering the Cluster** page appears, Choose an IP Address that is on the same Network as both Tentacles and a hostname that is 15 characters or less.
+
+*![](/docs/images/clustered-listening-tentacles/configure-clusterhostname.jpg)*
+
+Now complete the Wizard.
+
+## Adding Octopus Tentacle as a Generic Service Cluster {#ClusteringTentacles-AddTentacleCluster}
+
+Right-Click **Roles** and select **Configure Roles** then Highlight **Generic Service**, click **Next**.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-servicetype.jpg)*
+
+Find and highlight **OctopusDeploy Tentacle** service in the list of available services, click **Next**
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-selectservice.jpg)*
+
+Under **Client Access Point** choose an apropriate NetBIOS Name and IP Address for this clustered role. Note down this IP Address/DNS Hostname, you will be using it in the Octopus Server for connection to the Tentacle Cluster.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-clientaccess.jpg)*
+
+Under **Select Storage** choose the Disk which we had mounted previously and installed the Home and Application folders to previously
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-storage.jpg)*
+
+Under **Replication Registry Settings** Add a new root registry key of "Software\Octopus" and complete the wizard.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-newrolewizard-key.jpg)*
+
+You have now successfully completed the installation of Octopus Deploy Tentacles in a new Active/Passive Cluster, in listening mode.
+
+*![](/docs/images/clustered-listening-tentacles/cluster-complete.jpg)*
+
+## Connect Octopus Server to a Clustered Tentacle {#ClusteringTentacles-ConnectOctopusServer}
+
+Note down the IP Address which was specified in the **Client Access Point** Page of the new Role wizard. In this example, the IP Address was 192.168.1.206
+
+Login to the Octopus Server and go to the **Environments** Page. Under the desired Environment, click **Add Deployment Target**.
+
+For the target type, choose **Listening Tentacle**.
+
+For the hostname, enter the IP Address of the Clustered Role, in this example it was 192.168.1.206 then click "Discover". 
```

Comment:
Not sure what a `clustered role` is, but not sure its needed here - you can just say `For the hostname, enter the IP you noted down earlier, then click "Discover". `